tools/plugin_tests/utils.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
tools is meant for complete standalone tools, not for build/nox helpers.
build_helpers it a good place for such logic.

Was there a reason that you merged it without waiting for review?

Reply:
> tools is meant for complete standalone tools, not for build/nox helpers.
> build_helpers it a good place for such logic.
> 
> Was there a reason that you merged it without waiting for review?

I didn't merge this :) 
I created this and then realized that I probably shoudn't do this.

Reply:
thx for the pointer, I will checkout out `build_helpers`

Reply:
we could move build helpers to tools, but right now you are starting to spread operational utilities in too many places.